src/index.ts

```diff
@@ -90,6 +90,7 @@ export interface IApplicationConfig {
   readonly spidLevelsWhitelist: ReadonlyArray<keyof SPID_LEVELS>;
   readonly startupIdpsMetadata?: Record<string, string>;
   readonly eventTraker?: EventTracker;
+  readonly hasDeltasLogging?: boolean;
```

Comment:
Should be preferable a more meaningful name.
Es. `hasClockSkewLoggingEvent`

Reply:
done in https://github.com/pagopa/io-spid-commons/pull/121/commits/2bdab2b11ec25400417c4303bdbc0e17f83f211c